config-solaris.md

```diff
@@ -90,7 +90,7 @@ Mapped to `allowed-address` in the [zonecfg(1M)][zonecfg.1m_2] man page.
 * **`configureAllowedAddress`** *(string, OPTIONAL)* If configureAllowedAddress is set to true, the addresses specified by allowedAddress are automatically configured on the interface each time the container starts.
 When it is set to false, the allowedAddress will not be configured on container start.
 Mapped to `configure-allowed-address` in the [zonecfg(1M)][zonecfg.1m_2] man page.
-* **`defrouter`** *(string, OPTIONAL)* The value for the OPTIONAL default router.
+* **`defaultRouter`** *(string, OPTIONAL)* The value for the OPTIONAL default router.
```

Comment:
It seems inconsistent to me to expand `def` → `default` here but [*not* expand `Priv` → `Privilege`][1].  I personally prefer the abbreviated names, but am not sure how the Solaris folks want to handle this.  I'm fine with this expansion and/or the `Priv` → `Privilege` expansion happening in a follow-up PR if we want to isolate those changes from the camelCase changes.

[1]: https://github.com/opencontainers/runtime-spec/pull/859#issue-231471046